compiler/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ include_directories(${GSLLITE_INCLUDE_DIRS})
 include_directories(${CHAINER_COMPILER_ROOT_DIR})
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/..)
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
+include_directories(${CUDA_INCLUDE_DIRS})
```

Comment:
Unnecessary if we use runtime/meminfo instead?

Reply:
When I tried to use runtime/memnifo, I encountered an issue that `g_meminfo_enabled` may be `false` and I cannot call `GetMemoryUsageInBytes()` properly. If it is ok to set `g_meminfo_enabled=true` before calling `GetMemoryUsageInBytes()`, then I think I can change this part.